Assets/Scripts/GameManagement/DataPersistence/FileDataHandler.cs

```diff
@@ -61,6 +62,7 @@ public void WriteObjectToJson(object data)
         {
             string dataToStore = JsonUtility.ToJson(data, true);
             fileWriter.WriteLine(dataToStore);
+            Debug.Log(dataToStore);
```

Comment:
probably remove/comment this out before merging

Reply:
This was something specific that Saran requested to allow for a little more transparency when saving/loading